Sources/Pokepay/BankAPI/User/BankPayTopUp.swift

```diff
@@ -27,7 +29,8 @@ public extension BankAPI.User {
             let dict: [String: Any] = [
                 "account_id": accountId,
                 "bank_id": bankId,
-                "amount": amount
+                "amount": amount,
+                "request_id": requestId
```

Comment:
Will have to conditionally set it instead.
```
if requestId != nil {
    dict["request_id"] = requestId
}
```

Reply:
and I fixed this one too.